free-programming-books.md

```diff
@@ -1767,6 +1767,8 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 * [PHP Programming](https://en.wikibooks.org/wiki/PHP_Programming)
 * [PHP Security Guide](http://phpsec.org/projects/guide/)
 * [PHP: The Right Way](http://www.phptherightway.com)
+* [PHPUnit Manual](https://phpunit.de/manual/current/en/phpunit-book.pdf) - Sebastian Bergmann (PDF)
+* [PHPUnit Pocket Guide](http://www.avveofficial.com/download/phpunit-pocket-guide.pdf) - Sebastian Bergmann (PDF)
```

Comment:
This is not a permissible source for an O'Reilly book. Look at other O'Reilly links in this list for the correct format.

Reply:
@eshellman I tried getting a link on O'Reilly, but seems this is not a free one.
I think i will have to remove this link